src/main/java/employee/data/EmployeeMongo.java

```diff
@@ -0,0 +1,114 @@
+package employee.data;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import javax.persistence.Id;
+
+@Document(collection = "employees")
+public class EmployeeMongo {
+
+    @Id
+    @Field("employeeId")
```

Comment:
Does it work if you add these Mongo specific annotations to the regular `Employee` pojo ?